ProjectLighthouse/Types/Entities/Level/SlotEntity.cs

```diff
@@ -29,6 +29,9 @@ public class SlotEntity
 
     public string IconHash { get; set; } = "";
 
+    [NotMapped]
+    public string InfoXml => $"<slot type=\"{this.Type}\" id=\"{this.SlotId}\" icon=\"{this.IconHash}\">{this.Name}</slot>";
```

Comment:
More documentation here might reduce confusion in the future for other contributors